iso15118/secc/controller/simulator.py

```diff
@@ -860,155 +796,33 @@ async def get_ac_charge_params_v20(
-            ),
+            evse_max_charge_power=RationalNumber.get_rational_repr(30000),
+            evse_max_charge_power_l2=RationalNumber.get_rational_repr(30000),
+            evse_max_charge_power_l3=RationalNumber.get_rational_repr(30000),
+            evse_min_charge_power=RationalNumber.get_rational_repr(100),
+            evse_min_charge_power_l2=RationalNumber.get_rational_repr(100),
+            evse_min_charge_power_l3=RationalNumber.get_rational_repr(100),
+            evse_nominal_frequency=RationalNumber.get_rational_repr(50),
+            max_power_asymmetry=RationalNumber.get_rational_repr(0),
+            evse_power_ramp_limit=RationalNumber.get_rational_repr(100),
+            evse_present_active_power=RationalNumber.get_rational_repr(0),
+            evse_present_active_power_l2=RationalNumber.get_rational_repr(0),
+            evse_present_active_power_l3=RationalNumber.get_rational_repr(0),
         )
         if selected_service == ServiceV20.AC:
             return ac_charge_parameter_discovery_res_params
         elif selected_service == ServiceV20.AC_BPT:
             return BPTACChargeParameterDiscoveryResParams(
                 **(ac_charge_parameter_discovery_res_params.dict()),
-                evse_max_discharge_power=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_max_discharge_power  # noqa
-                ),
-                evse_max_discharge_power_l2=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_max_discharge_power_l2  # noqa
-                ),
-                evse_max_discharge_power_l3=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_max_discharge_power_l3  # noqa
-                ),
-                evse_min_discharge_power=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_min_discharge_power  # noqa
-                ),
-                evse_min_discharge_power_l2=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_min_discharge_power_l2  # noqa
-                ),
-                evse_min_discharge_power_l3=RationalNumber.get_rational_repr(
-                    self.evse_data_context.rated_limits.ac_bpt_limits.evse_min_discharge_power_l3  # noqa
-                ),
+                evse_max_discharge_power=RationalNumber.get_rational_repr(30000),
+                evse_max_discharge_power_l2=RationalNumber.get_rational_repr(30000),
+                evse_max_discharge_power_l3=RationalNumber.get_rational_repr(30000),
+                evse_min_discharge_power=RationalNumber.get_rational_repr(100),
+                evse_min_discharge_power_l2=RationalNumber.get_rational_repr(100),
+                evse_min_discharge_power_l3=RationalNumber.get_rational_repr(100),
```

Comment:
Now that we have the context conveniently defined, in a different PR, we could replace the simulated numbers with those from the context.

Reply:
for the charge parameters one we need the real deal... is from the call to get_ac_charge_params or get_dc_charge_params that the data in the context is originated from. 
So, for the simulator we need some kind of data...